src/EventStore.Core/TransactionLog/Scavenging/Interfaces/IIndexReaderForAccumulator.cs

```diff
@@ -0,0 +1,19 @@
+﻿using EventStore.Core.Services.Storage.ReaderIndex;
+
+namespace EventStore.Core.TransactionLog.Scavenging {
+	public interface IIndexReaderForAccumulator<TStreamId> {
+		//qq review: consider whether to have maxposition / positionlimit instead of scavengepoint?
```

Comment:
+1 for `long beforePosition` instead of `scavengePoint`

Reply:
i actually decided to leave this as is (unless you feel strongly about it), the scavengepoint is still restricted to the scavenging area of the code

Reply:
nope, sounds good